scss/detail.scss

```
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.titulo-libro{
+  color: black;
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+}
+
+.titulo-genero{
+  color: gray;
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+  margin: 0 0 55px;
+}
+
+.titulo-secundario{
+  color: black;
+  font-family: 'Roboto', sans-serif;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.secundario-detalle{
+  color: gray;
+  font-family: 'Roboto', sans-serif;
+  font-size: 20px;
+  line-height: 24px;
+  margin: 0 0 36px 0;
```

You can use a shorthand here
```suggestion
  margin: 0 0 36px;
```